### PR TITLE
Add AlexGames matrix widget

### PR DESCRIPTION
**Pull request by alexbarry**
AlexGames is a collection of Lua and Rust games which can be built to web assembly and run in a browser. My original version uses a websocket server, but I created a prototype version that works as a matrix widget, see https://matrix.org/blog/2024/09/27/this-week-in-matrix-2024-09-27/#alexgames-simple-lua-rust-webassembly-powered-board-games-running-in-a-matrix-widget for more info.

Also, I included this link in my description, but it extends beyond 80 characters. I truncated the rest of the lines. Let me know if you'd rather I remove the link, I added it to my project's Github README, and it's in the Matrix room description.

### Checklist

- [x] Use an informative PR title, such as "Add foo to bar"
- [x] Format each entry as follows, where "Repo" and "Chat" are examples of
  optional extra links:
```
- [Name](https://example.com/) - Short description, sentence case. ([Repo](https://github.com/foo/bar), [Chat](https://matrix.to/#/#room:example.com)) `License` `Language`
```
- [x] Append to the bottom of a section
- [x] Wrap lines to 80 characters
- [x] Use [SPDX license identifiers](https://spdx.org/licenses/) when possible

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,6 +237,12 @@ Research papers and similar documents studying something related to Matrix.
   `Apache-2.0` `TypeScript`
 - [matrix-widget-debug](https://github.com/turt2live/matrix-widget-debug) - Test
   widget for validating clients and `matrix-widget-api`. `Apache-2.0` `JavaScript`
+- [AlexGames](https://github.com/alexbarry/AlexGames) - A simple board games
+  web app that works as a Matrix widget, see
+  ["This Week in Matrix" 2024-09-27](https://matrix.org/blog/2024/09/27/this-week-in-matrix-2024-09-27/#alexgames-simple-lua-rust-webassembly-powered-board-games-running-in-a-matrix-widget)
```
Comment (jryans) [inline README.md:242]: Hmm, I would think most of the info from that TWIM post would also be available at a homepage or repo, and that does appear to be the case with this repo. The TWIM post then feels a bit redundant, so let's remove that.

File: README.md [hunk 2]
```diff
@@ -237,6 +237,12 @@ Research papers and similar documents studying something related to Matrix.
...
+  for more information. ([Repo](https://matrix.to/#/#alexgames:matrix.org),
```
Comment (jryans) [inline README.md:243]: The repo is already the primary link above, so this repo link can be removed.

**Review by jryans — CHANGES_REQUESTED** [review]
Thanks for addition, seems like a interesting effort.

A few tweaks mentioned below, then it should be ready to go.

**Comment by alexbarry** [comment]
Done! I added the TWIM article to my README after updating this pull request, you're right that it's a bit redundant now.

**Review by jryans — COMMENTED** [review]
Ah, looks like you also removed the project chat room too... I didn't mean to imply that should also be removed, it would be fine to keep that if you'd like to.

Either way is fine, just let me know what you'd prefer.

**Comment by alexbarry** [comment]
> Ah, looks like you also removed the project chat room too... I didn't mean to imply that should also be removed, it would be fine to keep that if you'd like to.

Oops, my mistake. In a hurry I pressed "delete to next bracket", thinking it was "(Repo) (Chat)", not realizing it was "(Repo, Chat)".  I didn't think you wanted it removed.

Thanks for catching it! I put it back in.

**Review by jryans — APPROVED** [review]
Looks good, thanks! 😄

**Pull request merged**